docs-ref-mapping/reference-unified.yml

```diff
@@ -770,6 +770,11 @@
       children:
       - 'azure-mgmt-synapse'
       - 'azure-synapse'   
+  - name: Tables
+    uid: azure.python.sdk.landingPage.services.Tables
+    href: ~/docs-ref-services/tables-readme.md
```

Comment:
Per the current automation, this should be `data-tables-readme.md`

We just peel `azure-` or `@azure` or `azure.` off the beginning of package id and add `-readme`. I'll add an example directly talking about this in the docs primer.